create-release.sh

```diff
@@ -18,3 +19,18 @@ cd "$( cd "$( dirname "$0" )" && pwd )" || { echo "ERROR: Could not find the loc
 # Zip everything together
 zip -r flypie@schneegans.github.com.zip common daemon presets resources schemas settings locale && \
 zip -r flypie@schneegans.github.com.zip -- *.js metadata.json *.md LICENSE
+
+# Check whether the extension should be installed
+while getopts i FLAG; do
+	case $FLAG in
+		
+		i)  # shellcheck disable=2015
+        gnome-extensions install flypie@schneegans.github.com.zip --force && \
+        echo "Successfully installed the application! Now restart the Shell ('Alt'+'F2', then 'r')." || \
+        { echo "ERROR: Could not install the extension."; exit 1; }
+        rm flypie@schneegans.github.com.zip;;
```

Comment:
I chose to delete the zip file, since we only want to install the extension.
The user that runs the script downloaded the zip in the first place - no need to keep it then ;)

For actual zip file creation, just run the script without any arguments.